tools/pd-simulator/main.go

```diff
@@ -45,6 +45,7 @@ var (
 	caseName                    = flag.String("case", "", "case name")
 	serverLogLevel              = flag.String("serverLog", "fatal", "pd server log level")
 	simLogLevel                 = flag.String("simLog", "fatal", "simulator log level")
+	simLogFile                  = flag.String("simLogFile", "", "simulator log file")
```

Comment:
do we need to add default value?

Reply:
It is previously to stdout, so the behavior is as before.